<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QWidget" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserWidget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	
	font: 9pt &quot;微软雅黑&quot;;
}
QWidget#widget_current
{
	
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>600</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget_current" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>当前用户:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_userName">
           <property name="text">
            <string>user</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>类型:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_userType">
           <property name="text">
            <string>type</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableView" name="tableView_user"/>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>操作</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>15</number>
            </property>
            <item>
             <widget class="QPushButton" name="pushButton_addUser">
              <property name="text">
               <string>添加用户</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_changePwd">
              <property name="text">
               <string>修改密码</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_deleteUser">
              <property name="text">
               <string>删除用户</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_addAdmin">
              <property name="text">
               <string>添加管理员</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>72</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>75</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
